packages/autocomplete-core/src/__tests__/concurrency.test.ts

```diff
@@ -15,6 +15,7 @@ describe.skip('concurrency', () => {
       return defer(() => {
         return [
           {
+            sourceId: 'testSource',
```

Comment:
If we rather use `createSource` here, we don't need to worry about giving it a `sourceId`.

(Same comment applies at different occurrences.)